### ISSUE TITLE
In version 4.12.0, it often freezes.

### ISSUE THREAD

**Issue by starFelix** [ProxymanApp/Proxyman#1803]
## Description
In version 4.12.0, it often freezes.

## Steps to Reproduce

After using for a period of time, it is highly likely to occur without any specific operation, just running in the background.

## Current Behavior

it freezed.

## Expected Behavior

it should works normally.

## Environment

- App version: e.g Proxyman 4.12.0
- macOS version: e.g macOS 13.6

**Comment by NghiaTranUIT** [comment]
May I ask:
- In the Tools Menu -> SSL Proxying List -> How many items in the Include List? Does it have wildcard items, such as `*` for all traffic?
- If the app is often frozen, how much RAM does it uses? 🤔 You can get the RAM Usage in the right bottom bar.

**Comment by starFelix** [comment]
> May I ask:
> 
> * In the Tools Menu -> SSL Proxying List -> How many items in the Include List? Does it have wildcard items, such as `*` for all traffic?
> * If the app is often frozen, how much RAM does it uses? 🤔 You can get the RAM Usage in the right bottom bar.

for Q1: yes. there is a wildcard items in my proxying list.
for Q2: I will reply you when it happen again.

**Comment by NghiaTranUIT** [comment]
You might remove the `*` wildcard in the SSL Proxying List. This wildcard means Proxyman will try to decrypt all HTTPS requests/responses from your machine. It can consume a lot of RAM when you watch a Youtube video for example.

Remove it will resolve the problem/

**Comment by starFelix** [comment]
> You might remove the `*` wildcard in the SSL Proxying List. This wildcard means Proxyman will try to decrypt all HTTPS requests/responses from your machine. It can consume a lot of RAM when you watch a Youtube video for example.
> 
> Remove it will resolve the problem/

Oh, I misunderstood your question. I am not using a single wildcard *, but rather using a wildcard to match a part of a domain name.

**Comment by starFelix** [comment]
![image](https://github.com/ProxymanApp/Proxyman/assets/5734060/98e1a29c-8eec-4bff-bef5-295d114676e7)

**Comment by starFelix** [comment]
> You might remove the `*` wildcard in the SSL Proxying List. This wildcard means Proxyman will try to decrypt all HTTPS requests/responses from your machine. It can consume a lot of RAM when you watch a Youtube video for example.
> 
> Remove it will resolve the problem/

and It didnt happen on old versions.

**Comment by fredschnek** [comment]
Yes, I'm having the same issues here. I'm on Sonoma 14.0. I can no longer use the App, it simple does not work anymore when it's not freezing... It was perfect until 4.10.X
